webclient/projekt/views.py

```diff
@@ -1617,25 +1617,24 @@ def katastr_text_to_id(request):
     hlavni_katastr: str = request.POST.get("hlavni_katastr")
     if hlavni_katastr is None or hlavni_katastr.strip() == "":
         return request.POST.copy()
-    hlavni_katastr_name = hlavni_katastr[: hlavni_katastr.find("(")].strip()
-    okres_name = (
-        (hlavni_katastr[hlavni_katastr.find("(") + 1 :]).replace(")", "").strip()
-    )
-    katastr_query = RuianKatastr.objects.filter(
-        Q(nazev=hlavni_katastr_name) & Q(okres__nazev=okres_name)
-    )
+    try:
+        kod=hlavni_katastr[hlavni_katastr.find(';')+1:hlavni_katastr.find(')')].strip()
+    except Exception as e:
```

Comment:
Já bych doporučil odchytávat konkrétní chybu a ne obecnou `Exception`, pokud je to možné.